commit 13a92bfbd534d9f237f38b048be2740c26f6749e (refs/changes/31/2131/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-06-23T11:18:51-07:00 (2 years, 4 months ago)
    
    MANTA-3317 upload-mako-ls uses all memory, then uploads incomplete dump
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>
    Reviewed by: Kody A Kantor <kody.kantor@gmail.com>
    Approved by: Brian Bennett <brian.bennett@joyent.com>
    Approved by: Kody A Kantor <kody.kantor@gmail.com>

diff --git a/bin/upload_mako_ls.sh b/bin/upload_mako_ls.sh
index 92b4582..c3c5eee 100755
--- a/bin/upload_mako_ls.sh
+++ b/bin/upload_mako_ls.sh
@@ -33,7 +33,7 @@ DIR_TYPE='application/json; type=directory'
 LOG_TYPE='application/x-bzip2'
 PID=$$
 PID_FILE=/tmp/upload_mako_ls.pid
-TMP_DIR=/tmp/mako_dir
+TMP_DIR=/var/tmp/mako_dir
 LISTING_FILE=$TMP_DIR/$MANTA_STORAGE_ID
 MANTA_DIR=/mako
 
